crates/ruff/src/rules/pyflakes/snapshots/ruff__rules__pyflakes__tests__f841_dummy_variable_rgx.snap

```diff
@@ -256,4 +256,22 @@ F841_0.py:127:21: F841 Local variable `value` is assigned to but never used
     |
     = help: Remove assignment to unused variable `value`
 
+F841_0.py:152:25: F841 [*] Local variable `_` is assigned to but never used
+    |
+150 |     try:
+151 |         pass
+152 |     except Exception as _:
+    |                         ^ F841
+153 |         pass
+    |
+    = help: Remove assignment to unused variable `_`
+
+ℹ Fix
+149 149 | def f():
+150 150 |     try:
+151 151 |         pass
+152     |-    except Exception as _:
+    152 |+    except Exception:
+153 153 |         pass
+
```

Comment:
(This is expected -- it's a variant of the fixture where we replace the dummy variable regex with `z`.)